packages/glob/_test.pony

```diff
@@ -99,14 +99,14 @@ class iso _TestFilter is UnitTest
   fun name(): String => "files/Glob.filter"
 
   fun apply(h: TestHelper) ? =>
-    let m = Glob.filter([ "12/a/Bcd", "12/q/abd", "34/b/Befd"], "*/?/B*d")
+    let m = Glob.filter([ "12/a/Bcd"; "12/q/abd"; "34/b/Befd"], "*/?/B*d")
```

Comment:
I know it wasn't introduced in this PR, but this seems like a good opportunity to clean up the style inconsistency here for the space following the `[`, which isn't consistent with other single-line arrays in the standard library.